package.json

```diff
@@ -13,6 +13,7 @@
   "license": "Apache-2.0",
   "gypfile": true,
   "dependencies": {
+    "always-tail": "^0.2.0",
```

Comment:
since the tail is injected in on-core, so the `awlays-tail` is not necessary here.